cluster.go

```diff
@@ -627,7 +627,7 @@ func validateOptions(opts Options) error {
 				return err
 			}
 		} else {
-			_, err = libcni.ConfFromBytes([]byte(opts.Kubelet.CNIConfFile.Content))
+			_, err = libcni.NetworkPluginConfFromBytes([]byte(opts.Kubelet.CNIConfFile.Content))
```

Comment:
`libcni.ConfFromBytes()` is deprecated.
https://github.com/containernetworking/cni/blob/v1.3.0/libcni/conf.go#L274